src/Extism.Pdk/Interop.cs

```diff
@@ -79,7 +82,7 @@ public static void SetOutput(MemoryBlock block)
     /// Set the output data to be sent back to the host as a byte buffer.
     /// </summary>
     /// <param name="data">The byte buffer to set as output data.</param>
-    public unsafe static void SetOutput(ReadOnlySpan<byte> data)
+    public static unsafe void SetOutput(ReadOnlySpan<byte> data)
```

Comment:
interesting, out of curiosity are these order independent?

Reply:
nope, i will reverse the change